pull_request_template.md

```diff
@@ -1,17 +1,11 @@
-## Purpose
+## Description
 > Describe the problems, issues, or needs driving this feature/fix and include links to related issues in the following format: Resolves issue1, issue2, etc.
 
-## Goals
-> Describe the solutions that this feature/fix will introduce to resolve the problems described above
+## Type of change
 
-## Approach
-> Describe how you are implementing the solutions. Include an animated GIF or screenshot if the change affects the UI (email documentation@wso2.com to review all UI text). Include a link to a Markdown file or Google doc if the feature write-up is too long to paste here.
+- [ ] Change is only applicable for Developer view
+- [ ] Change is only applicable for Platform Engineer view
+- [x] Change is applicable for both Developer and Platform Engineer views
 
-## Release note

```

Comment:
shall we add a checklist part too :D. It kinda forces you to double check intrinsically.

Reply:
Checklist as in?

Reply:
tested PE view
tested Developer view ?

Reply:
Ack. Added a testing checklist with [096f438](https://github.com/wso2/docs-choreo-dev/pull/1786/commits/096f438307efe6013292ccfb9c867d30e996abe9)